ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -2,7 +2,7 @@
 nginx_sites:
 - server:
    file_name: "{{ deploy_env }}_commcare"
-   listen: "443 ssl"
+   listen: "443 ssl {{ nginx_set_default_server }}"
```

Comment:
nitpicky but this variable name seems odd to me. (not sure where the `set` part comes from)